src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40 * days;
+  const shortPeriod = 3;
+  const longPeriod = 7;
```

Comment:
Better to show in what units you measure the period 👇
```suggestion
  const shortPeriodDays = 3;
  const longPeriodDays = 7;
```